internal/uvm/network.go

```diff
@@ -507,7 +507,7 @@ func (uvm *UtilityVM) RemoveEndpointsFromNS(ctx context.Context, id string, endp
 	return nil
 }
 
-// RemoveEndpointFromNS removes ``endpoint` in the network
+// RemoveEndpointFromNS removes “endpoint` in the network
```

Comment:
double `` was formatted to a quote for some reason

Reply:
Lovely